.github/workflows/sample-cicd.yaml

```diff
@@ -0,0 +1,61 @@
+s workflow will build and push a new container image to Amazon ECR,
```

Comment:
I think here should have a # to comment this line

Reply:
done!